src/core_plugins/tile_map/public/__tests__/geohash_layer.js

```diff
@@ -1,13 +1,14 @@
 import expect from 'expect.js';
 import { KibanaMap } from 'ui/vis/map/kibana_map';
 import { GeohashLayer } from '../geohash_layer';
-import { GeoHashSampleData } from './geohash_sample_data';
+// import { GeoHashSampleData } from './geohash_sample_data';
```

Comment:
Is geohash_sample_data.js even needed? Looks like it is no longer used.

Reply:
you're correct, this can be removed.